common/osu/osuapi.py

```diff
@@ -611,6 +658,43 @@ def __score_data_from_ossapi(
         else:
             raise ValueError(f"{gamemode} is not a valid gamemode")
 
+        statistics = {}
+
+        if score.statistics.miss is not None:
+            statistics["miss"] = score.statistics.miss
+        if score.statistics.meh is not None:
+            statistics["meh"] = score.statistics.meh
+        if score.statistics.ok is not None:
+            statistics["ok"] = score.statistics.ok
+        if score.statistics.good is not None:
+            statistics["good"] = score.statistics.good
+        if score.statistics.great is not None:
+            statistics["great"] = score.statistics.great
+        if score.statistics.perfect is not None:
+            statistics["perfect"] = score.statistics.perfect
+        if score.statistics.small_tick_miss is not None:
+            statistics["small_tick_miss"] = score.statistics.small_tick_miss
+        if score.statistics.small_tick_hit is not None:
+            statistics["small_tick_hit"] = score.statistics.small_tick_hit
+        if score.statistics.large_tick_miss is not None:
+            statistics["large_tick_miss"] = score.statistics.large_tick_miss
+        if score.statistics.large_tick_hit is not None:
+            statistics["large_tick_hit"] = score.statistics.large_tick_hit
+        if score.statistics.small_bonus is not None:
+            statistics["small_bonus"] = score.statistics.small_bonus
+        if score.statistics.large_bonus is not None:
+            statistics["large_bonus"] = score.statistics.large_bonus
+        if score.statistics.ignore_miss is not None:
+            statistics["ignore_miss"] = score.statistics.ignore_miss
+        if score.statistics.ignore_hit is not None:
+            statistics["ignore_hit"] = score.statistics.ignore_hit
+        if score.statistics.combo_break is not None:
+            statistics["combo_break"] = score.statistics.combo_break
+        if score.statistics.slider_tail_hit is not None:
+            statistics["slider_tail_hit"] = score.statistics.slider_tail_hit
+        if score.statistics.legacy_combo_increase is not None:
+            statistics["legacy_combo_increase"] = score.statistics.legacy_combo_increase
```

Comment:
this will be replaced when ossapi gives us access to the raw json https://github.com/tybug/ossapi/issues/88